tests/listobjects/listobjects.go

```diff
@@ -108,19 +109,27 @@ func runTests(t *testing.T, schemaVersion string, client ListObjectsClientInterf
 					require.NoError(t, err)
 				}
 
-				for _, assertion := range stage.Assertions {
+				for _, assertion := range stage.ListObjectAssertions {
 					// assert 1: on regular list objects endpoint
+					t.Logf("ListObject request: %s. Contextual tuples: %s", assertion.Request, assertion.ContextualTuples)
+
+					contextualTuples := &pb.ContextualTupleKeys{
```

Comment:
Please inline this below

```
ListObjects(ctx, &pb.ListObjectsRequest{
    ...
    ContextualTuples: &pb.ContextualTupleKeys{
        TupleKeys: assertion.ContextualTuples,
    },
})
```

Reply:
we no longer have contextualTuples variable